Holovibes/sources/gui/windows/panels/image_rendering_panel.cc

```diff
@@ -130,17 +130,13 @@ void ImageRenderingPanel::save_gui(json& j_us)
 
 void ImageRenderingPanel::set_image_mode(int mode)
 {
-    if (UserInterfaceDescriptor::instance().import_type_ == ImportType::None)
+    if (api::get_import_type() == ImportType::None)
         return;
 
     if (mode == static_cast<int>(Computation::Raw))
     {
         api::close_windows();
         api::close_critical_compute();
-
-        if (!UserInterfaceDescriptor::instance().is_enabled_camera_)
-            return;
-
```

Comment:
Is it really ok to remove this ?

Reply:
Yes because `is_enabled_camera_` before was used to tell whether there is calculation camera or import file. So know i have deleted it. But in fact after this PR i will add check in the API side to prevent viewing if no computation is done